### PR TITLE
hotfix 22.4.1 - disable dashboard timetable

### PR DESCRIPTION
**Pull request by adrianjost**
Ticket: https://ticketsystem.schul-cloud.org/browse/SC-3672

image

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5201887&bid=17062889)

File: views/dashboard/dashboard.hbs [hunk 1]
```diff
@@ -17,7 +17,9 @@
         </div>
     </div>
 
-    <div class="timetable">
+	<p class="my-2">Der Stundenplan ist derzeit ausgeblendet. Gehe stattdessen zu Kurse, um deine Lerninhalte zu finden.</p>
```
Comment (Parnswir) [inline views/dashboard/dashboard.hbs]:
```suggestion
	<p class="my-2">Der Stundenplan ist derzeit ausgeblendet. Gehe stattdessen in den Kursbereich, um deine Lerninhalte zu finden.</p>
```

**Comment by konstantin-sc** [comment]
👍max text is better

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5201887&bid=17062889)

**Comment by Parnswir** [comment]
PR Code Quality können wir hier ignorieren. Betrifft nur die Kommentarzeilen

**Pull request merged**